Massdrop/fluxible-router
# Explicitly save scroll position across popstate navigation.

Explicitly save the scroll position that should be restored on popstate navigation, since an unknown entity can overwrite the window.history.state value during this process. Also do not overwrite this value ourselves if we are still processing a navigation process.

To test:
* In your massdrop-react repo, temporarily reference this change by updating the package.json 'fluxible-router' entry to point to: "git@github.com:Massdrop/fluxible-router.git#bac78dfbfdedb5d3650bc4e3b139ab0974604906".
* Load up the site, navigate around, navigation should be as normal.
